test/install-argo.sh

```diff
@@ -25,7 +25,7 @@ ACCOUNT=$(gcloud info --format='value(config.account)')
 kubectl create clusterrolebinding PROW_BINDING --clusterrole=cluster-admin --user=$ACCOUNT --dry-run -o yaml | kubectl apply -f -
 kubectl create clusterrolebinding DEFAULT_BINDING --clusterrole=cluster-admin --serviceaccount=default:default --dry-run -o yaml | kubectl apply -f -
 
-ARGO_VERSION=v2.7.5
+ARGO_VERSION=v2.11.6
```

Comment:
Not sure if I need to update this one

Reply:
I think this will be needed for all the test infrastructure to run with argo 2.11.6